code/modules/power/apc.dm

```diff
@@ -658,16 +658,16 @@
 	else if (istype(W, /obj/item/device/debugger))
 		if(emagged || hacker || infected)
 			to_chat(user, "<span class='warning'>There is a software error with the device. Attempting to fix...</span>")
-			if(do_after(user, 10 SECONDS, act_target = src))
+			if(do_after(user, 10/W.toolspeed SECONDS, act_target = src))

```

Comment:
Lines 661, 663, 665, instead of "X seconds / toolspeed" have changed to "X ticks / toolspeed"

Reply:
Apologies I appear to be blind on this one